Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -538,8 +545,8 @@ namespace WhiteBox
         // temporary setting to disable this feature
         if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get())
         {
-            bool hasTransformMode = false;
-            settingsRegistry->Get(hasTransformMode, WhiteBoxTransformFeature);
+            bool hasTransformMode = true;
+            // settingsRegistry->Get(hasTransformMode, WhiteBoxTransformFeature);
```

Comment:
update: Would be better to just add this setting to your setreg file if possible?

Reply:
I should do that, not sure why I did this because I have it already in the json.